lib/core/src/server/manager/manager-webpack.config.js

```diff
@@ -58,9 +58,44 @@ export default ({ configDir, configType, entries, dll, outputDir, cache, babelOp
       new webpack.DefinePlugin({ 'process.env': stringified }),
       new CaseSensitivePathsPlugin(),
       new Dotenv({ silent: true }),
+      // graphql sources check process variable
+      new DefinePlugin({
+        process: JSON.stringify(true),
+      }),
+      // See https://github.com/graphql/graphql-language-service/issues/111#issuecomment-306723400
+      new ContextReplacementPlugin(/graphql-language-service-interface[/\\]dist/, /\.js$/),
     ].filter(Boolean),
     module: {
-      rules: [babelLoader(babelOptions)],
+      rules: [
```

Comment:
why????

Reply:
GraphiQL needs some CSS, and CSS might have local assets.

I do not think it's a bad thing to support this, also in light of people writing addons with their own preferences, they might want to add some css or file refs without having to change the manager webpack config.